charts/kubeclarity/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.kubeclarity.prometheus.serviceMonitor.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "kubeclarity.name" . }}
+  {{- with .Values.kubeclarity.prometheus.serviceMonitor.annotations }}
```

Comment:
you are missing a placeholder for `annotations` in `values.yaml`

Reply:
fixed